hellowgit/src/main/mule/hellowgit.xml

```diff
@@ -1,14 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<mule xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns="http://www.mulesoft.org/schema/mule/core"
-	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
-http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd">
+<mule xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
 	<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" doc:id="9318ca86-613f-462f-ba57-fb2803d46646" >
 		<http:listener-connection host="0.0.0.0" port="8081" />
 	</http:listener-config>
 	<flow name="hellowgitFlow" doc:id="2da7e852-7516-487e-a810-3ca94c87e8d3" >
 		<http:listener doc:name="Listener" doc:id="68ddeeef-cf8d-4525-a40c-e0264648527f" config-ref="HTTP_Listener_config" path="/git"/>
-		<set-payload value="Hello Git" doc:name="Set Payload" doc:id="7b3c74e7-5df8-4c9d-b4e3-d551a2dc6180" />
+		<ee:transform doc:name="Transform Message" doc:id="7ab6a00b-5709-4ceb-a5f5-e4a1280428f4" >
+			<ee:message >
+				<ee:set-payload ><![CDATA[%dw 2.0
+output application/json
+---
+{
+	message : payload
```

Comment:
what will be the payload here?

Reply:
the payload in this case will be hello world and git ,for the future it is gonna be user input.